hydradx/tests/test_omnipool_amm.py

```diff
@@ -2608,9 +2772,9 @@ def test_swap_exploit(lp_multiplier, trade_mult, oracle_mult):
 
 @given(
     omnipool_reasonable_config(),
-    st.floats(min_value=0, max_value=0.02, exclude_min=True),
+    st.floats(min_value=1e-8, max_value=0.02),
```

Comment:
Does this prevent a rare error?

Reply:
Yes, I was getting an error when the random draw was choosing a value like 1e-300, and this change prevented it.